src/pages/en/guides/images.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Images
+description: Learn how to use images in Astro.
+i18nReady: true
+setup: |
+  import Since from '../../../components/Since.astro';
+---
+Astro provides several ways for you to use images inside of your projects.
+
+### `.astro` files
+
+Astro uses standard HTML `<img>` or JSX `<img />` elements to display images within your `.astro` files. 
+
+```astro
+---
+// src/pages/index.astro
+import rocket from './images/rocket.svg';
+---
+<img src="https://astro.build/assets/press/full-logo-light.png" alt="Astro">
+<img src="/stars.png" alt="A starry night sky."> <!-- stored in public -->
+<img src="./images/astronaut.jpg" alt="An astronaut."> <!-- stored in src -->
+<img src={rocket} alt="A rocketship in space."/>
+```
+
+### .`md` and `.mdx` files
+
+You can use standard Markdown `![]()` syntax in your `.md` and `.mdx` files, or HTML (in Markdown) or JSX (in MDX) if preferred. 
+
+
+```md
+// src/pages/post-1.md
+![An astronaut on the moon.](./images/astronaut.png)
+<img src="/stars.png" alt="A starry night sky.">
+```
+
+
+```mdx
+// src/pages/post-2.mdx
+---
+image: './images/rocket.svg'
+---
+![An astronaut on the moon.](./images/astronaut.png)
```

Comment:
```suggestion
![An astronaut on the moon.](/images/astronaut.png) <!-- stored in public -->
```

Reply:
Ooh, I meant for the first one to be a relative path, just like in the `.md` example, but do you think it's a better example of the difference if we now make this one an image in `public/`? To show that you can use that syntax for public images, too? @kevinzunigacuellar 

If so, then I'd proabably also change the `stars` one to now be the relative path. So, show them as opposites, to make it clear that it's not like one syntax is for public and the other is not.

Reply:
@sarah11918, I don't think relative paths work for default image notation `![]()`. I tried this using rc-3 yesterday in a `.mdx` file.

Reply:
@kevinzunigacuellar I have this working for me, with an image in public, accessing with the ![]() notation! Do you mind trying again? I appear to be on rc-3 as well...

Reply:
I see my mistake, I thought the image was relative to the file not to `src` 😅

Reply:
You might have been on to something after all! I'm doing more testing, and also in build mode... So, it's great you brought this to my attention!